test/libsolidity/lsp/references/import_aliases.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.8.0;
+
+import "../goto/lib.sol" as Imported;
+//                          ^ @ImportedDef
+
+contract C
+{
+    function one(uint lhs, uint rhs) public pure returns (uint result)
+    {
+        result = Imported.Lib.add(lhs, rhs + 123);
+        //       ^ @ImportedUse
+        //       ^^^^^^^^ @Imported1
+        //       ^^^^^^^^^^^^ @ImportedLib1
+    }
+    function two(uint lhs, uint rhs) public pure returns (uint result)
+    {
+        result = one(Imported.Lib.add(lhs, rhs), rhs);
+        //                    ^ @LibRef
+        //           ^^^^^^^^ @Imported2
+        //           ^^^^^^^^^^^^ @ImportedLib2
+    }
+}
+// ----
+// lib: @diagnostics 2072
+// -> textDocument/documentHighlight {
+//     "position": @ImportedDef
+// }
+// <- [
+//     { "kind": 2, "range": @Imported1 },
+//     { "kind": 2, "range": @Imported2 }
+// ]
+// -> textDocument/documentHighlight {
+//     "position": @ImportedUse
+// }
+// <- [
+//     { "kind": 2, "range": @Imported1 },
+//     { "kind": 2, "range": @Imported2 }
+// ]
+// -> textDocument/documentHighlight {
+//     "position": @LibRef
```

Comment:
Can you add some other stuff with the same names as these identifiers to show that only the references that have the same definition are being found? For example a function parameter, local variable, struct field, user type defined inside an interface, named parameter of a function pointer, etc.

Same in the other import test.